xarray_jax/__init__.py

```diff
@@ -530,8 +456,7 @@ def apply_ufunc(func, *args, require_jax=False, **apply_ufunc_kwargs):
   def wrapped_func(*maybe_wrapped_args):
     unwrapped_args = [unwrap(a, require_jax) for a in maybe_wrapped_args]
```

Comment:
Do we still need to unwrap here? I think it may not be needed as we should be able to assume here that `func` will return a valid array?

In that case, we may not need to wrap `apply_ufunc` at all here and could just leave behind a backwards-compatibility alias `apply_ufunc = xarray.apply_ufunc` ?